proto/cosmos/bank/v1beta1/genesis.proto

```diff
@@ -21,8 +21,13 @@ message GenesisState {
   repeated cosmos.base.v1beta1.Coin supply = 3
       [(gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins", (gogoproto.nullable) = false];
 
-  // denom_metadata defines the metadata of the differents coins.
+  // denom_metadata defines the metadata of the different coins.
   repeated Metadata denom_metadata = 4 [(gogoproto.nullable) = false];
+
+  // send_enabled defines the denoms where send is enabled or disabled.
+  //
+  // Since: cosmos-sdk 0.47
```

Comment:
We don't know what the next version will be, in fact, we're striving towards 1.0.

Reply:
According to [ADR 044: Guidelines for Updating Protobuf Definitions](https://github.com/cosmos/cosmos-sdk/blob/main/docs/architecture/adr-044-protobuf-updates-guidelines.md#2-non-msg-related-protobuf-definitions-may-have-new-fields-but-must-add-a-since-comment), this needs to be `0.47`.

Reply:
The point is to be specific and use something consistently:
* we are not 100% sure what will be the next release. We have already many unplanned point releases (0.41, 0.43, 0.45 ...).
* so let's use whatever makes sense , it can be 1.0, 0.47, XXX  - something that we can easily search and update when the next release will be clear. And be consistent. Searching for 0.46 will be confusing. Searching for 0.47 or v1.0 in proto is more precise I think.